src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -146,3 +146,30 @@ table.matrix th.group-all-groups-cell {
 .matrix-general-search {
     margin-bottom: 10px;
 }
+
+.matrix-header {
+    .col-sm-11 {
+        h1 {
+            display: flex;
+            align-items: flex-end;
```

Comment:
So that “Experiment matrix” will be aligned with the bottom row of internal_tags badges?

Reply:
Yeah, exactly.